midend/simplifyKey.h

```diff
@@ -142,27 +152,35 @@ class TableInsertions {
  * @post all complex table key expressions are replaced with a simpler expression.
  */
 class DoSimplifyKey : public Transform {
-    ReferenceMap*     refMap;
-    TypeMap*          typeMap;
-    KeyIsSimple*      key_policy;
+    ReferenceMap* refMap;
+    TypeMap* typeMap;
+    KeyIsSimple* key_policy;
     std::map<const IR::P4Table*, TableInsertions*> toInsert;
 
  public:
-    DoSimplifyKey(ReferenceMap* refMap, TypeMap* typeMap, KeyIsSimple* key_policy) :
-            refMap(refMap), typeMap(typeMap), key_policy(key_policy)
-    { CHECK_NULL(refMap); CHECK_NULL(typeMap); CHECK_NULL(key_policy); setName("DoSimplifyKey"); }
+    DoSimplifyKey(ReferenceMap* refMap, TypeMap* typeMap, KeyIsSimple* key_policy)
+        : refMap(refMap), typeMap(typeMap), key_policy(key_policy) {
+        CHECK_NULL(refMap);
+        CHECK_NULL(typeMap);
+        CHECK_NULL(key_policy);
+        setName("DoSimplifyKey");
+    }
     const IR::Node* doStatement(const IR::Statement* statement, const IR::Expression* expression);
 
     // These should be all kinds of statements that may contain a table apply
     // after the program has been simplified
-    const IR::Node* postorder(IR::MethodCallStatement* statement) override
-    { return doStatement(statement, statement->methodCall); }
-    const IR::Node* postorder(IR::IfStatement* statement) override
-    { return doStatement(statement, statement->condition); }
-    const IR::Node* postorder(IR::SwitchStatement* statement) override
-    { return doStatement(statement, statement->expression); }
-    const IR::Node* postorder(IR::AssignmentStatement* statement) override
-    { return doStatement(statement, statement->right); }
+    const IR::Node* postorder(IR::MethodCallStatement* statement) override {
```

Comment:
@ChrisDodd won't like this; you are taking 3 times more lines.
I don't think we should run clang-format blindly.